### PR TITLE
Fixes the analytics automated report job deploy error

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Fixes the analytics deploy job typo
- use env vars described in file
image
image

### CODE REVIEW

File: .github/workflows/deploy-apis-to-production.yml [hunk 1]
```diff
@@ -832,10 +832,10 @@ jobs:
       - name: Build and Push Job's Docker Image
         run: |
           cd src/analytics/jobs/reports
-          docker build --target=production --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-analytics-report-job:${{ needs.image-tag.outputs.build_id }} .
-          docker tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-analytics-report-job:${{ needs.image-tag.outputs.build_id }} ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-analytics-report-job:latest
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-analytics-report-job:${{ needs.image-tag.outputs.build_id }}
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-analytics-report-job:latest
+          docker build --target=production --tag ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-analytics-report-job:${{ needs.image-tag.outputs.build_id }} .
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:835]: [use env vars described in file](https://github.com/airqo-platform/AirQo-api/pull/1252/commits/cd2d0299b8849ab6ab05c8e4cd2537c59da518c0)

File: .github/workflows/deploy-apis-to-staging.yml [hunk 2]
```diff
@@ -1021,10 +1021,10 @@ jobs:
...
-          docker tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-analytics-report-job:${{ needs.image-tag.outputs.build_id }} ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-analytics-report-job:latest
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-analytics-report-job:${{ needs.image-tag.outputs.build_id }}
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-analytics-report-job:latest
+          docker build --target=staging --tag ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-analytics-report-job:${{ needs.image-tag.outputs.build_id }} .
+          docker tag ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-analytics-report-job:${{ needs.image-tag.outputs.build_id }} ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-analytics-report-job:latest
+          docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-analytics-report-job:${{ needs.image-tag.outputs.build_id }}
+          docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-analytics-report-job:latest
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:1027]: [use env vars described in file](https://github.com/airqo-platform/AirQo-api/pull/1252/commits/cd2d0299b8849ab6ab05c8e4cd2537c59da518c0)

**Review by Baalmart — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: ["go", "java", "javascript", "python"]
+        language: ["go", "javascript", "python"]
```
Comment (123MwanjeMike) [inline .github/workflows/codeql-analysis.yml:35]: [Remove Java check from codeql](https://github.com/airqo-platform/AirQo-api/pull/1252/commits/c9b60da3bfc12331b06e38dbd0b0f4774330654c)

**Comment by Baalmart** [comment]
I am about to merge this PR. Thanks @123MwanjeMike .

cc: @NoahNsimbe

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**